src/diffpy/labpdfproc/tools.py

```
@@ -172,15 +173,32 @@ def set_wavelength(args):
     Raises
     ------
     ValueError
-        Raised when input wavelength is non-positive
-        or if input anode_type is not one of the known sources.
+        Raised if:
+        (1) neither wavelength or anode type is provided,
```

I removed the muD requirement here to simplify the logic. I think we will set wavelength before setting muD, so maybe we can raise the error there when we look up the muD?